mods/ra/rules/vehicles.yaml

```diff
@@ -759,7 +759,7 @@ STNK:
 		InitialStanceAI: ReturnFire
 	Armament:
 		Weapon: APTusk
-		LocalOffset: 400,0,0
+		LocalOffset: 400,0,171
```

Comment:
This is still too far forwards:
<img width="217" alt="screen shot 2017-02-11 at 10 45 22" src="https://cloud.githubusercontent.com/assets/167819/22853184/43060000-f047-11e6-8707-6fa251729419.png">

Please change to `192,0,176`.